arch/x86/core/ia32/intstub.S

```diff
@@ -192,6 +179,16 @@ alreadyOnIntStack:
 	pop %eax
 #endif
 
+
+#if defined(CONFIG_TRACING_ISR)
+	/* Save these as we are using to keep track of isr and isr_param */
+	pushl	%eax
```

Comment:
I don't think EAX needs to be pushed here, it used to contain the interrupt handler argument but that got pushed onto the stack earlier

The CONFIG_EXECUTION_BENCHMARKING block is also wrong but that's eventually being removed.

The stench of copy-paste is strong here...have to be very careful about doing this in assembly as there is no compiler to fix redundancies